spec/strava/api/client/endpoints/gears/gear_spec.rb

```diff
@@ -3,20 +3,25 @@
 RSpec.describe 'Strava::Api::Client#gear', vcr: { cassette_name: 'client/gear' } do
   include_context 'API client'
   it 'returns gear' do
-    gear = client.gear(id: 'g3423618')
+    gear = client.gear(id: 'b2338517')
     expect(gear).to be_a Strava::Models::Gear
-    expect(gear.id).to eq 'g3423618'
+    expect(gear.id).to eq 'b2338517'
     expect(gear.resource_state).to eq 3
-    expect(gear.distance).to eq 380_939.0
-    expect(gear.distance_in_miles_s).to eq '236.7mi'
-    expect(gear.name).to eq 'adidas Supernova ST'
-    expect(gear.primary).to be true
-    expect(gear.brand_name).to eq 'adidas'
-    expect(gear.model_name).to eq 'Supernova ST'
-    expect(gear.description).to eq 'Marathon 2018.'
+    expect(gear.distance).to eq 54_349
+    expect(gear.distance_in_miles_s).to eq '33.77mi'
+    expect(gear.name).to eq 'Trek'
+    expect(gear.primary).to be false
+    expect(gear.brand_name).to eq 'Trek '
+    expect(gear.model_name).to eq 'Madrone'
+    expect(gear.description).to eq 'white'
+    expect(gear.frame_type).to eq 3
+    expect(gear.weight).to eq 9.1
+    expect(gear.retired).to be false
   end
   it 'returns gear by id' do
-    gear = client.gear('g3423618')
+    gear = client.gear('b2338517')
     expect(gear).to be_a Strava::Models::Gear
   end
 end
+# curl -H 'Accept: application/json' -H "Authorization: Bearer access_token_here " https://www.strava.com/api/v3/gear/b2338517
```

Comment:
Remove this. If you want to add something to CONTRIBUTING that explains how to do such changes that'd be great though. Can be a separate PR.

Reply:
git doesn't think file changed if only comments removed, so commited a "bogus" change, then removed "bogus" and commit again. Anyhow the curl tip comments now removed.